hs_core/hydroshare/hs_bagit.py

```diff
@@ -231,7 +298,7 @@ def create_bag(resource):
     :param resource: (subclass of AbstractResource) A resource to create a bag for.
     :return: the hs_core.models.Bags instance associated with the new bag.
     """
-    create_bag_files(resource)
+    create_bag_metadata_files(resource)
```

Comment:
noticed the method name ```create_bag_metadata_files()``` here is exactly the same as the caller method on line 114. This does not look right.